src/stories/Library/material-search/material-search.scss

```diff
@@ -0,0 +1,235 @@
+// Variables
+$_gin_xs_spacing: 0.5rem;
+$_gin_m_spacing: 1rem;
+$_gin_border_color: #8e929c;
+$_gin_font_weight_semibold: 525;
```

Comment:
:+1: Nice with these variables. The naming clarifies the relationship.

I wonder if it would be relevant to move these to a shared file where they can be used by more admin components.

Reply:
Potentially yes, i think if we do maybe we could look into a bit more what is needed more broadly